### PR TITLE
completely remove spongeML, workers ai instead

### PR DESCRIPTION
**Pull request by artifishvr**

### CODE REVIEW

**Review by daniwasonline — COMMENTED** [review]
Why not use the Workers AI SDK?

**Comment by artifishvr** [comment]
there's an sdk???? i didn't see one except for when you're using a worker

**Comment by daniwasonline** [comment]
oh it looks like the SDK's been deprecated, go on

**Comment by artifishvr** [comment]
> go on

> doesn't approve

**Comment by artifishvr** [comment]
>_<

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "spongegpt",
-  "version": "1.30.1",
```
Comment (daniwasonline) [inline package.json:3]: why are we versioning down here?

File: index.js [hunk 2]
```diff
@@ -86,20 +77,23 @@ async function getPronouns(userid) {
 }
 
 let lastMessage = "";
-let enableLocal = false;
+let initialHistory = [
```
Comment (daniwasonline) [inline index.js:80]: normalise using const with a constant!! (correct me if i'm wrong but i don't see any mutations of initialHistory, always use a const by default. just small nitpicks)

**Review by daniwasonline — CHANGES_REQUESTED** [review]
fix the versioning, otherwise this is 👍👍

i'd also say to move to const instead of let for any vars that you don't mutate, normalising const means that the semantic of "let" has a mutatible meaning in your code and it'll be a lot easier to find where that happens

File: package.json (hunk 1 above)
Comment (artifishvr) [reply package.json:3]: wdym? do i version wrong

**Comment by artifishvr** [comment]
thats really funny

File: package.json (hunk 1 above)
Comment (daniwasonline) [reply package.json:3]: wait im dyslexic and an idiot sorry

**Comment by artifishvr** [comment]
see this is why having an org where others can poke at my terrible work and suggest changes is good

**Comment by daniwasonline** [comment]
> see this is why having an org where others can poke at my terrible work and suggest changes is good

it's good!! we're also (sort of) following stacking guidelines as well so now we don't have to spend loads of time dealing with conflicts

**Review by daniwasonline — APPROVED** [review]
looks good!!

**Comment by daniwasonline** [comment]
so tempted to click close w/ comment right now for s&g

**Pull request merged**